theano/gpuarray/opt.py

```diff
@@ -1091,25 +1091,19 @@ def local_gpua_advanced_boolean_subtensor(op, context_name, inputs, outputs):
 @op_lifter([tensor.AdvancedIncSubtensor1])
 @register_opt2([tensor.AdvancedIncSubtensor1], 'fast_compile')
 def local_gpua_advanced_incsubtensor1(op, context_name, inputs, outputs):
-    context = get_context(context_name)
-    # This is disabled on non-cuda contexts
-    if context.kind != b'cuda':
-        return None
-
     x, y, ilist = inputs
 
     set_instead_of_inc = op.set_instead_of_inc
 
-    compute_capability = int(context.bin_id[-2])
-    if (compute_capability >= 2 and x.ndim == 1 and y.ndim == 0 and
```

Comment:
CUDA 7.0 is the minimum version we support and it doesn't work with devices that have compute capability below 2 so this check is redundant.

If there are other tests for cc 2.0+ we should also remove them.  Things that check for higher compute capabilities would need to stay.